__tests__/buildTranslationFiles.spec.ts

```diff
@@ -332,6 +333,14 @@ describe.each(formats)('buildTranslationFiles in %s', (fileFormat) => {
         createTranslations(config);
         assertPartialTranslation({ type, expected, fileFormat });
       });
+
+      it('should work with angular 14 inject', () => {
```

Comment:
Why do we need this as it's own test case? seems like it's covered by the previous test